crates/services/consensus_module/poa/src/service_test.rs

```diff
@@ -220,6 +225,13 @@ impl BlockSigner for FakeBlockSigner {
     }
 }
 
+#[derive(Clone)]
+struct FakeBlockProductionTrigger;
+
+impl TriggerBlockProduction for FakeBlockProductionTrigger {
```

Comment:
It would be nice if you also added a unit test to check that this feature works=)

Reply:
addressed in fda6b1e2bc